doc/source/serve/tutorials/batch.md

```diff
@@ -80,113 +93,95 @@ the maximum possible batch size that Ray Serve executes at once.
 :start-after: __doc_deploy_begin__
 ```
 
-## Deploy the Deployment
-Deploy the deployment by running the following through the terminal.
+## Deployment Options
+
+You can deploy your application in two ways:
+
+### Option 1: Deploying with the Serve CLI
 ```console
-$ serve run tutorial_batch:generator
+$ serve run tutorial_batch:generator --name "Text-Completion-App"
 ```
 
-Define a [Ray remote task](ray-remote-functions) to send queries in
-parallel. While Serve is running, open a separate terminal window, and run the 
-following in an interactive Python shell or a separate Python script:
+Note that the `--name` argument is optional. If you do not specify a name, the app will be named `default`.
+ Naming the app is useful if you are running multiple apps on the same machine, and 
+also for being able to create a new handle to the app.
```

Comment:
don't think we need these details in the batching tutorial

Reply:
we get the handle below from the return of `serve.run` right? do we need to get it dynamically?

```
handle: DeploymentHandle = serve.run(generator, name="Text-Completion-App")
```

Reply:
The query code below is meant to be run from a separate script. The other Serve tutorials follow that pattern.

Reply:
Got it. In that case, would use: `serve.get_app_handle("default")`

Reply:
and up to you if you want to leave the `--name` part in there and use custom name

Reply:
ok done :)